src/services/DAB.ts

```diff
@@ -0,0 +1,9 @@
+import { HttpAgent } from '@dfinity/agent';
+import { DABCollection, getAllNFTS } from '@psychedelic/dab-js';
+import { fetch } from 'react-native-fetch-api';
+
+export const getAllDabNFTS = async (): Promise<DABCollection[]> => {
+  const agent = new HttpAgent({ fetch });
```

Comment:
don't you need to also send `host: IC_URL_HOST`?
`const agent = new HttpAgent({ fetch, host: IC_URL_HOST });`

Reply:
@aleacevedo Do you know?

Reply:
Yep, he is right, my bad🤦🏽‍♂️.
it should be the same that https://github.com/Psychedelic/plug-mobile/blob/6d987630164db6c3584236230a38682d5fd4de37/src/utils/walletConnect.js#L132 and later we should remove the one on utils